### DISCUSSION TITLE
Any plans to support community forks of Moq after the recent data collection / privacy concerns?

### DISCUSSION THREAD

**Discussion by orc-jpad** [AutoFixture/AutoFixture#1410] [Q&A] [answered]
Hello,

I was wondering if you are you aware of or have any plans to support community forks of [devlooped/moq](https://github.com/devlooped/moq) for auto-mocking after the recent data collection / privacy concerns? See issues https://github.com/devlooped/moq/issues/1372 and https://github.com/devlooped/moq/issues/1374. Even if the issue has been addressed, there seems to be a general distrust of the author remaining.

**Comment by aivascu** [comment, marked as answer]
Hello @orc-jpad ,
Due to how NuGet manages package dependencies, it is not possible to alternate between different package variations of the same codebase. This means that if we move `AutoFixture.AutoMoq` to a fork package, it will break a lot of codebases. In other words `AutoFixture.AutoMoq` will continue to use Moq for as long as long as Moq itself will be supported or another library takes its place.

In terms of the Moq forks integrations, there are no plans at the moment. However, if one of the forks will gain enough traction in the community to motivate the existence and support of a new package by AutoFixture, then it is not out of the question.
